__tests__/removeSingleKeyNodes.test.ts

```diff
@@ -0,0 +1,108 @@
+import { describe, expect, it } from "@jest/globals";
+import { removeSingleKeyNodes } from "../src/lib/utils";
+
+describe("remove single key nodes", () => {
+  it("Handles empty object", () => {
+    const data = {};
+    const expected = {};
+
+    expect(removeSingleKeyNodes(data)).toEqual(expected);
+  });
+
+  it("array returns same array", () => {
+    const data = [1, 2, 3];
+    const expected = [1, 2, 3];
+
+    const output = removeSingleKeyNodes(data);
+    expect(output).toEqual(expected);
+  });
+
+  it("single node key nested inside complex object", () => {
+    const data = {
+      umbrella: {
+        responses: { data: [1, 2, 3] },
+        theHitcher: "put you in the picture",
+        tony: { harrison: "outrage" },
+      },
+    };
+
+    const expected = {
+      data: [1, 2, 3],
+      theHitcher: "put you in the picture",
+      harrison: "outrage",
+    };
+    const output = removeSingleKeyNodes(data);
+    expect(output).toEqual(expected);
+  });
+
+  it("Handles nested single key arrays", () => {
+    const data = { root: { branch: { leaf: [1, 2, 3] } } };
+    const expected = { leaf: [1, 2, 3] };
```

Comment:
I think this is a special case (when it's an object with 1 key-value where the value is an array) where we want it to output `[1, 2, 3]`

Reply:
Believe this comment hasn't been addressed - in my opinion showing it as a table with multiple rows is best here rather than as a table with 1 row with 1 column filled with JSON